crate/server/src/config/command_line/clap_config.rs

```diff
@@ -1,11 +1,15 @@
-use std::fmt::{self};
+use std::{
+    fmt::{self},
+    path::PathBuf,
+};
 
 use clap::Parser;
 use serde::{Deserialize, Serialize};
 
 use super::{HttpConfig, JwtAuthConfig, MainDBConfig, WorkspaceConfig};
-use crate::telemetry::TelemetryConfig;
+use crate::{error::KmsError, result::KResult, telemetry::TelemetryConfig};
 
+const KMS_SERVER_CONF: &str = "/etc/cosmian_kms/kms.toml";
```

Comment:
Rename to` DEFAULT_COSMIAN_KMS_CONF` ?

Reply:
done in 6dfc3facda5ff669b064c5ffaa798490e85de80e